.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: tests
+
+on:
+  [push, pull_request]
```

Comment:
Maybe change this into only run on push to main and PR to main?

Reply:
Now it runs it for every commit on every branch

Reply:
I'll change this to PR only as we already have main protected